include/mata/nfa/strings.hh

```diff
@@ -183,14 +183,14 @@ public:
      * @return A vector of segments for the segment automaton in the order from the left (initial state in segment automaton)
      * to the right (final states of segment automaton).
      */
-    const AutSequence& get_segments();
+    const std::vector<Nfa::Nfa>& get_segments();
 
     /**
      * Get raw segment automata.
      * @return A vector of segments for the segment automaton in the order from the left (initial state in segment automaton)
      * to the right (final states of segment automaton) without trimming (the states are same as in the original automaton).
      */
-    const AutSequence& get_untrimmed_segments();
+    const std::vector<Nfa::Nfa>& get_untrimmed_segments();
```

Comment:
I guess there is a reason, but why not to open namespace Nfa in such a code?

Reply:
We already use `using namespace Mata::Nfa;` in this code. The problem is that when you write `Nfa`, the compiler cannot know whether you mean `struct Nfa`, or `namespace Mata::Nfa` and fails to compile. Refer to #204.

Reply:
aha, another reason remove the name clash somehow then